sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -154,6 +155,13 @@ def _build_uri(address, entity):
     address += "/" + str(entity)
     return address
 
+def _get_backoff_time(retry_mode, backoff_factor, retried_times):
+    if retry_mode == RetryMode.Fixed:
+        backoff_value = backoff_factor
+    else:
+        backoff_value = backoff_factor * (2 ** retried_times)
+    return backoff_value
```

Comment:
@yunhaoling we also just log if the backoff_value is more than the max (on line 351), but don't actually retry. However, in azure-core, it looks like they still retry with the [max_backoff if this backoff_value is greater ](https://github.com/Azure/azure-sdk-for-python/blob/35d5436cf58a207613cc2bfd1fa796c67f92af1e/sdk/core/azure-core/azure/core/pipeline/policies/_retry.py#L121)than allowed. Why don't we do this here?

Reply:
good catch, I think this is a bug, we should keep retrying as the core does